.github/workflows/test.main.kts

```diff
@@ -117,17 +132,28 @@ private fun validateTypings() {
         { it.owner == "DamianReeves" && it.name == "write-file-action" },
     )
 
+    val actions = System.getenv("changed_typings").let {
```

Comment:
Let's add a piece of log to both branches (checking only changed vs. all typings), especially the list of modified typings.

Reply:
We can add additional logging, but there already is logging.
If you look at https://github.com/typesafegithub/github-actions-typing-catalog/actions/runs/14842175031/job/41667555071#step:4:5 you see the value of the `changed_typings` environment variable, which is the list of modified or added typings, the `yaml` ones are then in the error message and the `yml` ones that are checked are still logged additionally with `➡️ For ...`.
So I'm not sure what additional logging you strive for.

Reply:
I was expecting a piece of log straight from the Kotlin-based logic, to ensure the logic went the correct path. What you added is good, thanks 👍